.tributors

```diff
@@ -0,0 +1,1422 @@
+{
+    "sappelhoff": {
+        "name": "Stefan Appelhoff",
+        "email": "stefan.appelhoff@mailbox.org",
+        "blog": "https://www.stefanappelhoff.com",
+        "orcid": "0000-0001-8002-0877",
+        "affiliation": "Max Planck Institute for Human Development"
+    },
+    "dungscout96": {
+        "name": "Dung Truong",
+        "blog": "https://github.com/dungscout96"
+    },
+    "effigies": {
+        "name": "Christopher J. Markiewicz",
+        "email": "effigies@gmail.com",
+        "blog": "https://github.com/effigies"
+    },
+    "tsalo": {
+        "name": "Taylor Salo",
+        "email": "tsalo006@fiu.edu",
+        "bio": "I am a Psychology Ph.D. student at FIU working with @angielaird in the @NBCLab. I am interested in performing reproducible, transparent research with fMRI.",
+        "blog": "https://tsalo.github.io",
+        "orcid": "0000-0001-9813-3167",
+        "affiliation": "University of Pennsylvania, Department of Psychiatry, Philadelphia, Pennsylvania, 19104, United States"
+    },
+    "chrisgorgo": {
+        "name": "Chris J. Gorgolewski",
+        "email": "krzysztof.gorgolewski@gmail.com",
+        "blog": "http://chrisgorgolewski.org",
+        "orcid": "0000-0003-3321-7583",
+        "affiliation": "Google LLC"
+    },
+    "franklin-feingold": {
+        "name": "Franklin W. Feingold",
+        "bio": "BIDS: https://github.com/bids-standard/bids-specification\r\n\r\nOpenNeuro: https://github.com/OpenNeuroOrg/openneuro ",
+        "orcid": "0000-0002-6533-2909",
+        "affiliation": "Stanford University ",
+        "blog": "https://github.com/franklin-feingold"
+    },
+    "Remi-Gau": {
+        "name": "Remi Gau",
+        "email": "remi.gau@tuebingen.mpg.de",
+        "bio": "Neuroimaging (high-res & laminar fMRI) and multisensory.\r\n\r\nToo much matlab and presentation and not enough R and python",
+        "blog": "https://remi-gau.github.io/",
+        "orcid": "0000-0002-1535-9767",
+        "affiliation": "Université catholique de Louvain"
+    },
+    "yarikoptic": {
+        "name": "Yaroslav O. Halchenko",
+        "email": "debian@onerussian.com",
+        "bio": "Cheers!",
+        "blog": "http://www.onerussian.com",
+        "orcid": "0000-0003-3456-2493",
+        "affiliation": "Dartmouth College"
+    },
+    "TheChymera": {
+        "name": "Christian Horea",
+        "email": "chr@chymera.eu",
+        "bio": "I bring Reproducibility, Freedom, and Open Source to Neuroscience, Academic Publishing, Software, and Life.",
+        "blog": "http://www.chymera.eu",
```

Comment:
```suggestion
        "blog": "http://www.chymera.eu",
        "orcid": "XXXX-XXXX-XXXX-XXXX",
```

Reply:
@TheChymera the orcid number you had given in the google form was missing some digits.

Do you think you could complete it here?

Thanks

Reply:
@Remi-Gau I went to his paper and followed an ORCiD link: https://orcid.org/0000-0001-7037-2449

Reply:
@Remi-Gau [0000-0001-7037-2449](https://orcid.org/0000-0001-7037-2449) is indeed correct. Also my website has https — should I sumbit a PR for this?

Reply:
don't worry: just enough to tell me here.

Addd an "http://" to some of the "url" people put in the form because otherwise the citation.cff file is invalid according to their schema. But I forgot to check whether after doing so the URL was actually valid. 🤦🏾